pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
+
+`printf "header1 header2\nbar foo\n" | column -c {{10}}`
+
+- Specify characters to use to delimit columns for the -t option (i.e. "," for csv), default is whitespace:
+
+`printf "header1,header2\nbar,foo\n" | column -s{{,}}`
+
+- Determine the number of columns and auto-align the output in a tabular format:
```

Comment:
As I see it, letting the command detect columns without providing a separator is indeed automatic. By passing `-s`, the user explicitly tells the command where to split the lines, which dictates how many columns there are. I don't see how that can be described as automatic. So I'd refrain from mentioning `-s` here. I'd just say something like "Split columns automatically, using whitespace as column separators".